internal/provider/kubernetes/controller.go

```diff
@@ -1204,7 +1204,7 @@ func (r *gatewayAPIReconciler) watchResources(ctx context.Context, mgr manager.M
 	return nil
 }
 
-func (r *gatewayAPIReconciler) enqueueManagedClass(obj client.Object) []reconcile.Request {
+func (r *gatewayAPIReconciler) enqueueManagedClass(_ context.Context, obj client.Object) []reconcile.Request {
```

Comment:
curious why this API signature had to change ?

Reply:
see https://github.com/kubernetes-sigs/controller-runtime/pull/2139